### DISCUSSION TITLE
反代ip扫描思路怎么做到的

### DISCUSSION THREAD

**Discussion by romanholidaypancakes** [XIU2/CloudflareSpeedTest#400] [提问] [answered]
ip非常快 可惜不更新了

https://github.com/ip-scanner/cloudflare/issues/8

**Comment by XIU2** [comment, marked as answer]
和 CloudflareST 的 HTTPing 原理一样，不过毕竟不是专业的扫描软件，因此在扫描用途上 功能、效率 可能不如专门的扫描软件。

你说的这个项目，就是先找一些常见 IDC 的 IP 段，然后遍历 IP 去访问某个使用了 Cloudflare CDN 的网站，如果能访问就说明该 IP 服务器上建立了端口转发 或 SNI 代理。

**Comment by mengguyi** [comment]
或许你可以试试把cfst里带的ip.txt换成阿里巴巴cloud的IP然后扫描试试？

**Comment by charSLee013** [comment]
不良林以前有发过[视频](https://www.youtube.com/watch?v=uKXVXaa5_YI)讲过这些，本质就是没加过滤，自建的反代Cloudflare的IP地址
